src/features/security-cloud-game/SecurityCloudGame.js

```diff
@@ -129,10 +128,10 @@ const SecurityCloudGame = ({ height, intl: { formatMessage }, onValidDrop, width
         setGameState(prevState => ({ ...prevState, isOverlap: isOverlapping }));
 
         if (shouldUpdateLiveText) {
-            const newliveText = isOverlap
+            const newliveText = isOverlapping
```

Comment:
do we need the transform on 128?

minor but I think `hasOverlap` might be better. but also not necessary to change

Reply:
It's the recommended way to update state object. However, it does seem a bit redundant as we are only updating one property at a time in each `setGameState` call, so I broke up the gameState object into individual states, which also looks much cleaner.

Changed to `hasOverlap`.

(trying to hit 100 comments lol)

Reply:
oh sorry I actually misread it. I was referring to:
`setGameState(prevState => ({ ...prevState, isOverlap: isOverlapping }));`
would actually be:
`setGameState(prevState => ({ ...prevState, isOverlap }));`

so that the change to `const newliveText = isOverlapping` was not needed. but I realize that wouldn't work since `isOverlap` was already a variable in the outer scope.

Reply:
uh my bad, I misinterpreted. But yeah, I had to use a different name cuz `isOverlap` was taken already.